src/main/java/edu/aau/groupc/canteenbackend/user/User.java

```diff
@@ -0,0 +1,76 @@
+package edu.aau.groupc.canteenbackend.user;
+
+import edu.aau.groupc.canteenbackend.entities.DBEntity;
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Data
+@Entity
+@Table(name = "users", uniqueConstraints = {
+        @UniqueConstraint(columnNames = {"username"}),
+})
+public class User implements DBEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String username;
+    private String password;
```

Comment:
Like I mentioned last review, consider not storing the password in plain text.
You could hash the password in the conversion process from input DTO to the DB Entity. 
(See [this tutorial section](https://www.baeldung.com/java-password-hashing#3-generating-a-salt) for SHA-512 in Java.)
Then, just changing the data type from `String` to `byte[]` inside the Entity should work for this purpose.

However, this is not striclty required for the functionality of the system, so only do if you can manage with reasonable effort.